ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss

```diff
@@ -1,10 +1,14 @@
 @import "../app-theme.scss";
 
 .dashboard-title-section {
-    padding: 2rem 0;
+    padding: 2rem 0 0 0;
     background-color: mat-color($app-theme, 350);
 }
 
+.dashboard-section {
+    margin-bottom: 10rem;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/30126907/71064053-30b5b780-217f-11ea-8845-cca509424454.png)

Reply:
The added spacing looks good.
Noticing that if you narrow the page, the status icon don't line up:
<img width="669" alt="Screen Shot 2019-12-22 at 1 47 51 PM" src="https://user-images.githubusercontent.com/29984380/71325939-07b85e00-24c2-11ea-8442-a4b8b26055fb.png">